### PR TITLE
python3Packages.just-start: init at 2018-10-25

### PR DESCRIPTION
**Pull request by dtzWill**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/just-start/default.nix [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    sha256 = "0vm3jp3a8nibmbgik2daq4wb437qrc2ri2zwb3c49rnl6y7dw258";
+  };
+
+  propagatedBuildInputs = [ pexpect urwid toml pydantic ];
+
+  checkInputs = [ glibcLocales ];
+
+  LC_ALL = "C.UTF-8";
+
+  # Fails with message complaining about our fake $HOME
```
Comment (dotlambda) [inline pkgs/development/python-modules/just-start/default.nix]: You can set
```
preCheck = ''
  export HOME=$(mktemp -d)
'';
```

File: pkgs/development/python-modules/just-start/default.nix [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  doCheck = false;
+
+  meta = with lib; {
+    description = "An app to defeat procrastination (terminal pomodoro w/taskwarrior)";
```
Comment (dotlambda) [inline pkgs/development/python-modules/just-start/default.nix]: If it's an application, why put it in `pythonPackages`?

**Review by dotlambda — COMMENTED** [review]
According to https://github.com/samuelcolvin/pydantic/blob/v0.18.2/setup.py#L52, a `disabled = !isPy3k` is missing.

**Comment by FRidh** [comment]
Nothing happening so closing.

**Pull request closed without merging**